async-clipboard/demo.js

```diff
@@ -0,0 +1,55 @@
+let copy = document.querySelector('#copy');
+let paste = document.querySelector('#paste');
+let out = document.querySelector('#out');
```

Comment:
prefer `const` to `let`

Reply:
it'll take me an eternity to get used to this